packages/radix-ui-themes/changelog.md

```diff
@@ -7,6 +7,18 @@
   - Remove comments from the CSS build.
   - Make sure that forced light/dark appearance on the `Theme` component also sets the corresponding browser colors, like the correct input autofill background color.
   - Use `outline` rather than `box-shadow` for most focus styles, which avoids a slight anti-aliasing issue in Chrome on focused elements
+- `TextArea`, `TextField`
+  - Refine padding values for a more balanced look.
+    - Use `text-indent` instead of `padding-left` so that long values aren't truncated on the left when the cursor is at the end of the input.
```

Comment:
Left edge not clipped:

<img width="368" alt="Screenshot 2023-09-04 at 12 35 08" src="https://github.com/radix-ui/themes/assets/8441036/de0b00a8-d332-47c9-8541-da0948a1fe74">